.github/workflows/ubuntu_20.04/build.sh

```diff
@@ -51,7 +51,9 @@ CXXFLAGS="-std=c++17 -O1 $ARCH_FLAGS" CFLAGS="-O1 $ARCH_FLAGS" ./configure --pre
     --with-kea=/usr/bin/kea-config \
     --with-tiledb \
     --with-crypto \
-    --with-ecw=/opt/libecwj2-3.3
+    --with-ecw=/opt/libecwj2-3.3 \
+    --with-zarr \
+    --enable-driver-zarr
```

Comment:
this should not be necessary with a correct configure script, --with-zarr should be sufficient to activate the driver

Reply:
:+1: